api/src/test/java/com/pip/chatbot/test/SampleTest.java

```diff
@@ -0,0 +1,12 @@
+package com.pip.chatbot.test;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
```

Comment:
To uruchamia cały springowy context, wystarczy nam test jednostkowy. Wywalmy @SpringBootTest